frontend/src/components/Typography/Body/index.js

```diff
@@ -6,7 +6,7 @@ const Body = ({ children }) => {
 };
 
 Body.propTypes = {
-  children: PropTypes.string.isRequired,
+  children: PropTypes.string,
```

Comment:
Is it not required?

Reply:
So there is a case where we might pass something to the body that is null depending on another thing.

Reply:
You made the error go away, but you didn't actually fix the core issue. Which is that an `<Input>` component was used without providing a `name`. It's `undefined`.

Reply:
Okay, on it!

Reply:
Done!

Reply:
Now that I think about it, with Joao's PR this error is probably gone anyways :thinking: